code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -372,4 +372,46 @@
 /obj/item/weapon/storage/belt/fannypack/yellow
  	name = "yellow fannypack"
  	icon_state = "fannypack_yellow"
- 	item_state = "fannypack_yellow"
\ No newline at end of file
+ 	item_state = "fannypack_yellow"
+
+/obj/item/weapon/storage/belt/sabre
+	name = "sabre sheath"
+	desc = "An ornate sheath designed to hold an officer's blade. Alt click to draw it quickly."
+	icon_state = "sheath"
+	item_state = "sheath"
+	storage_slots = 1
+	w_class = 4
+	max_w_class = 4
+	attack_verb = list("struck", "bashed", "disciplined")
+	use_sound = null
+	can_hold = list(
+		/obj/item/weapon/material/sword/captain_sword
+		)
+
+/obj/item/weapon/storage/belt/sabre/AltClick(mob/user)
+	if(!usr || usr.stat || usr.lying || usr.restrained() || !Adjacent(usr))

```

Comment:
Use of `usr` in a non-interface proc. No, refactor to user. And use `use_check(user)`.